Makefile

```diff
@@ -163,24 +167,25 @@ container-push:
 # download controller-gen if necessary
 controller-gen:
 ifeq (, $(shell command -v controller-gen ;))
-	@echo "controller-gen not found in PATH, checking in GOBIN ($(GOBIN))..."
-ifeq (, $(shell command -v $(GOBIN)/controller-gen ;))
-	$(GO_CMD) install sigs.k8s.io/controller-tools/cmd/controller-gen@v0.6.2
-	@echo "controller-gen installed in GOBIN"
+	@echo "controller-gen not found in PATH, checking $(GOBIN_ALT)"
+ifeq (, $(shell command -v $(GOBIN_ALT)/controller-gen ;))
+	@GOBIN=$(GOBIN_ALT) GO_CMD=$(GO_CMD) hack/install-tools.sh --controller-gen
```

Comment:
Just to be on the safe side: `$(CURDIR)/hack/install-tools.sh` ?

Reply:
ACK

Reply:
(and we can even do it better ;)